shared/src/test/diff/mlscript/Annoying.mls

```diff
@@ -71,14 +71,14 @@ def test x = case x of { int -> x.f | _ -> x.x }
 //│ ╔══[ERROR] Type mismatch in def definition:
 //│ ║  l.57: 	def test x = case x of { int -> x.f | _ -> x.x }
```

Comment:
It seems like the ^^^^^ should be highlighting part `_ -> x.x`

Reply:
Good catch! The locations of `case` and `def` trees were off. Fixed in https://github.com/hkust-taco/mlscript/pull/74/commits/6ccdf208c629e22c96e2dfadf50dadda188b3c70